packages/CLAPACK/meta.yaml

```diff
@@ -33,4 +33,6 @@ build:
 
     emmake make -j ${PYODIDE_JOBS:-3} blaslib lapacklib
     mkdir -p dist/lib
-    emcc blas_WA.a lapack_WA.a F2CLIBS/libf2c.a -sSIDE_MODULE -o dist/lib/clapack_all.so
+    emcc blas_WA.a lapack_WA.a F2CLIBS/libf2c.a -sSIDE_MODULE=1 -o dist/lib/clapack_all.so
+    emcc blas_WA.a -sSIDE_MODULE=1 -o dist/lib/libblas.so
+    emcc lapack_WA.a -sSIDE_MODULE=1 -o dist/lib/liblapack.so
```

Comment:
This is going to create trouble because whenever CLAPACK is loaded all of the lapack and blas symbols will be linked twice. I'm not sure what the emscripten dynamic linker will do with all these duplicate symbols, but it seems worrisome. If suitesparse doesn't directly use dlopen then it should be okay to put these into e.g. `build/lib` and pass `-L$(PACKAGE_ROOT)/CLAPACK/build/lib` to suitesparse. At runtime, the symbols will come from `clapack_all.so` but they are the same symbols.